### PR TITLE
Created the Places package and configurations

### PR DESCRIPTION
**Pull request by 701i**
fixes issue #4

### CODE REVIEW

**Review by emanuelalvaradog — APPROVED** [review]
Everything seems ok

**Comment by 701i** [comment]
Completed 5/6 tasks of my issue #4, the last it's configuration for the CSS preprocessor and I'm investigating it

File: packages/places/.storybook/main.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "../src/**/*.stories.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions"
+  ],
+  "framework": "@storybook/react"
+}
```
Comment (gh0stl1m) [inline packages/places/.storybook/main.js:12]: EOF (end of file)

**Review by emanuelalvaradog — APPROVED** [review]
👌

**Review by gh0stl1m — APPROVED** [review]

**Pull request merged**